### ISSUE TITLE
NVDA reads the subject more than once before it starts reading the message in Mozilla Thunderbird and Seamonkey Mail

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1776]
**Reported by Shah on 2011-09-12 10:43**
Starting with NVDA snapshot Main 4658, when opening a message in Thunderbird 6, NVDA reads the subject at least 2 or three times before it starts reading the message. Thunderbird is set to open messages in a separate window.

**Comment by bhavyashah** [comment]
@jcsteh For some odd reason, I happen to believe that you do use Mozilla Thunderbird. Could you and other NVDA+Thunderbird users, as well as NVDA+SeaMonkey Mail users, if any, please test and inform us if this bug still persists?

**Comment by jcsteh** [comment]
I don't use Thunderbird any more.

**Comment by bdorer** [comment]
Yes, this issue still occours but only if new messages are opened in a 
ne window, not in a tab.
Here's a short log fragment showing you what's NVDA telling after 
opening a message:
Input: kb(laptop):NVDA+l
IO - speech.speak (21:57:21.553):
Speaking [u'Ausgeklappt   Re: [nvaccess/nvda] NVDA reads the subject 
more than once before it starts reading the message in Mozilla 
Thunderbird and Seamonkey Mail (#1776)   Leonard de Ruijter 06:18']
IO - inputCore.InputManager.executeGesture (21:57:31.071):
Input: kb(laptop):enter
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole 
(21:57:31.671):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 
0, None))
IO - speech.speak (21:57:31.723):
Speaking [u'[nvaccess/nvda] NVDA reads the subject more than once before 
it starts reading the message in Mozilla Thunderbird and Seamonkey Mail 
(#1776) - Mozilla Thunderbird']
IO - speech.speak (21:57:31.726):
Speaking [u'[nvaccess/nvda] NVDA reads the subject more than once before 
it starts reading the message in Mozilla Thunderbird and Seamonkey Mail 
(#1776) - Mozilla Thunderbird']
IO - speech.speak (21:57:31.730):
Speaking [u'Unbekannt']
IO - speech.speak (21:57:31.786):
Speaking [u'[nvaccess/nvda] NVDA reads the subject more than once before 
it starts reading the message in Mozilla Thunderbird and Seamonkey Mail 
(#1776) - Mozilla Thunderbird']
IO - speech.speak (21:57:31.790):
Speaking [u'Re: [nvaccess/nvda] NVDA reads the subject more than once 
before it starts reading the message in Mozilla Thunderbird and 
Seamonkey Mail (#1776)']
IO - speech.speak (21:57:31.793):
Speaking [IndexCommand(1), u'@jcsteh', u' For some odd reason, I happen 
to believe that you do use Mozilla Thunderbird. ']

I tested this with Thunderbird 52.2.1 (32-Bit) and NVDA next 
next-14290,07052e9d


Am 07.08.2017 um 06:18 schrieb bhavyashah:

**Comment by Adriani90** [comment]
@jcsteh do you have any updates on this issue?

**Comment by jcsteh** [comment]
Unfortunately, I don't. I don't use Thunderbird any more personally, and Thunderbird isn't developed by Mozilla Corporation any more, so I can't really spend time digging into this. That said, it'd be interesting to know whether this still happens in Thunderbird beta or nightlies.

**Comment by shah** [comment]
@Adriani90 I'm not a Thunderbird user so I'm unable to confirm.

**Comment by bdorer** [comment]
This is still reproducable with latest Thunderbird and NVDA next

**Comment by Adriani90** [comment]
This is still reproducible in NVDA alpha-19420,a2c6f0f8
Thunderbird 68.3.0 and Windows 10 1909 update (build 18363.47).

**Comment by Adriani90** [comment]
This is still reproducible with NVDA 2024.4 Beta and Thunderbird 128.2.0, the subject is read 5 or 6 times on my end befor NVDA start reading the current line of the message body.
In contrast, in MS Outlook NVDA reads toe subject only once after pressing enter on the message, then NVDA starts reading the first line of the message body.
cc: @SaschaCowley
